draft-ietf-acme-acme.md

```diff
@@ -2506,27 +2506,29 @@ Initial contents:
 \[\[ RFC EDITOR: Please replace XXXX above with the RFC number assigned to this
 document ]]
 
-### Challenge Types {#iana-challenge}
+### Validation Methods {#iana-challenge}
 
-This registry lists the ways that ACME servers can offer to validate control of
-an identifier.  The "Identifier Type" field in the template must be contained
-in the Label column of the ACME Identifier Types registry.
+This registry lists identifiers for the ways that CAs can validate control of
+identifiers.  Each method's entry must specify whether it corresponds to an
+ACME challenge type.  The "Identifier Type" field must be contained in the
+Label column of the ACME Identifier Types registry.
 
 Template:
 
-* Label: The value to be put in the "type" field of challenge objects using this
-  validation mechanism
-* Identifier Type: The type of identifier that this mechanism applies to
-* Reference: Where the challenge type is defined
+* Label: The identifier for this validation method
+* Identifier Type: The type of identifier that this method applies to
+* ACME: "Y" if the validation method corresponds to an ACME challenge type;
```

Comment:
In the current spec, you don't need IETF consensus to add to the challenge types registry.  So this change would be raising the bar.

In any case, what I want this column to capture is not some process point, only whether someone has defined how to do this method with ACME.